_people/bakos-gabor.markdown

```diff
@@ -1,8 +1,16 @@
 ---
-title: Bakos Gabor
+title: Gabor Bakos
 date: 2018-05-01 22:25:00 Z
 Member Type:
   Is Voting Member: true
+Photo: http://www.fragosus.com/portfolio/img/portfolio/gbakos.jpg
```

Comment:
@fragosus Instead of linking to an external site for the photo, can you add this to the `/uploads` folder on your repo? If you need help doing this, we can definitely help out.

Reply:
I have no push access to the repository, so yes I would really appreciate your help...

Reply:
@fragosus You should have push access to your own repo, or do you not? 

You can add the image to the `_uploads` folder on your repo: https://github.com/fragosus/hotosm-website/tree/gh-pages/_uploads, then link replace the Photo metadata item with `/uploads/gbakos.jpg`.